plugin/src/sbt-test/paradox/snippets/src/main/paradox/group.md

```diff
@@ -0,0 +1,3 @@
+@@ snip [Group A](../../test/scala/Multiple.scala) { #multiple-something-else group=a }
+
+@@ snip [Group B](../../test/scala/Multiple.scala) { #parseint-imports group=b }
```

Comment:
Added a selectbox for switching in #107 

Actually selecting tabs/alternatives is done in javascript, so isn't easy to incorporate in a test...